dbt_style_guide.md

```diff
@@ -531,111 +531,156 @@ when needed.
   ) %}
   ```
 
-## Metrics style guide
+## Semantic Models & Metrics style guide
+
+## Organizing Semantic Model
+
+- Semantic models are created in the same yml file where you define configs, tests and metrics for your models. There should be one yml file per model.

```

Comment:
Use the same capitalization of YAML that appears throughout the rest of the file:

```suggestion
- Semantic models are created in the same YAML file where you define configs, tests and metrics for your models. There should be one YAML file per model.

```